test/test_go.rb

```diff
@@ -1924,7 +1875,7 @@ def test_file_completion
     tmux.until { |lines| lines.count { |l| l.include?('.fzf-home') } > 1 }
     tmux.send_keys :Enter
     tmux.until(true) do |lines|
-      lines[-1].end_with?('.fzf-home')
+      lines[-1] =~ %r{cat .*/\.fzf-home}
```

Comment:
Nice. Thanks to `-J`, right?

Reply:
Thanks! `lines[-1].end_with?('.fzf-home')` is supposed to confirm that fzf put the selected item on the shell's command line, but in fact it matches the last fzf result (when the query `'.fzf-home` has one result). The test is then liable to send subsequent keys (`cat ~such**` etc.) before the shell is ready, causing this intermittent failure: https://travis-ci.org/github/jablko/fzf/jobs/673032619#L928-L974

[Line 1934](https://github.com/jablko/fzf/blob/298eda3806811964cc31f6c381a4df3b8c3ddfe7/test/test_go.rb#L1932-L1934) sent `cat ~such**` and is waiting for fzf to start, but the shell never saw those keys and still shows `cat /home/travis/.fzf-home`.

`%r{cat .*/\.fzf-home}` fixes the test so it confirms what it should, which resolves the intermittent failure by not sending subsequent keys too quickly.

Reply:
Ah, thanks for the explanation. Yep, that makes sense.